### DISCUSSION TITLE
Allow to customize the redirect pages

### DISCUSSION THREAD

**Discussion by ahqsoftwares** [withastro/roadmap#927] [Proposal]
### Body

# Summary
Basically allow the devs to make their custom redirect pages so that it looks consistent without a sudden white flickering

# Goals
- To customize the page

**Comment by artt** [comment]
I'm quite new to this discussion thing, but I think this is a great idea. How do we go on from this? Can I just create a PR or does it need some kind of mod approval?

Relevant code: https://github.com/withastro/astro/blob/main/packages/astro/src/core/routing/3xx.ts

**Reply by ahqsoftwares** [reply]
Probably you'll just need to pr

**Reply by ematipico** [reply]
@artt it's all written in the readme of this repository: https://github.com/withastro/roadmap

This isn't a small feature, as it requires:
- code
- testing 
- docs
- requirements 
- discussion 

Just opening a PR won't be enough

**Reply by artt** [reply]
Thanks. I thought I've seen this somewhere but couldn't find it.

On another note: what's the difference between a roadmap and a feature request? (I've seen issues being tagged as feature request, but not in this repo.) Or does it depend on repo?
